### DISCUSSION TITLE
[Script request] 2FAuth

### DISCUSSION THREAD

**Discussion by di5cord20** [community-scripts/ProxmoxVE#160] [Request script]
### Application Name

2fauth

### Website

https://github.com/Bubka/2FAuth

### Description

2FAuth is a web based self-hosted alternative to One Time Passcode (OTP) generators like Google Authenticator, designed for both mobile and desktop.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by jkrgr0** [comment]
Related PR: #943
